test/mixer/audio_mixers_test.exs

```diff
@@ -0,0 +1,428 @@
+defmodule Membrane.AudioMixersTest do
```

Comment:
I am not sure if I like the idea of merging tests for both mixers

* the code is too complex
* when you want to debug one mixer, you almost always need to run two pipelines - one for AudioMixer and the other one of LiveAudioMixer

Reply:
theoretically live audio mixer should pass all tests that an offline audio mixer does. This approach forces future developers to check if added functionality also works for live audio mixer.